src/renderer/common/redux/sagas/browse.ts

```diff
@@ -18,12 +18,12 @@ import { apiSaga } from "./api";
 const filename_ = "readium-desktop:renderer:redux:saga:opds-browse";
```

Comment:
I think that this TypeScript file should keep its original name `opdsBrowse.ts`, as the exported function is called `opdsBrowse()` and its `debug()` prefix is `readium-desktop:renderer:redux:saga:opds-browse`.

Reply:
done